src/editors/components/EditorFooter/index.test.jsx

```diff
@@ -68,14 +68,14 @@ describe('EditorFooter', () => {
       createdCallback();
       expect(saveBlock).toHaveBeenCalled();
       expect(sendTrackEvent).toHaveBeenCalledTimes(1);
-      expect(sendTrackEvent).toHaveBeenCalledWith(analytics.editorSaveClick, props.analytics);
+      expect(sendTrackEvent).toHaveBeenCalledWith(analyticsEvt.footer.editorSaveClick, props.analytics);
     });
     test('handleCancelClicked calls navigateCallback', () => {
       sendTrackEvent.mockClear();
       realmodule.handleCancelClicked({ returnUrl: props.returnUrl });
       expect(navigateCallback).toHaveBeenCalledWith(props.returnUrl);
```

Comment:
The current code does not pass this test as this is a callback within a callback. This will be adjusted when adjusting to Ben's changes for the video editor.